src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -1002,10 +991,16 @@ dmaTest(const std::shared_ptr<xrt_core::device>& _dev, boost::property_tree::ptr
     logger(_ptTest, "Details", (boost::format("Buffer size - '%s'") % xrt_core::utils::unit_convert(block_size)).str());
 
     // check if the bank has enough memory to allocate
-    //  m_size is in KB so convert block_size (bytes) to KB for comparision
+    // m_size is in KB so convert block_size (bytes) to KB for comparison
     if(mem.m_size < (block_size/1024))
       continue;
 
+    size_t totalSize = 0;
```

Comment:
This code is functional and works for this release.  
Going forward, you might wish to refactore the code and add a set of helper routines in a common header file that will perform a cleaner conversion.

I came across the following code when reviewing a PR from Soren:
`constexpr size_t operator"" _kb(unsigned long long v)  { return 1024u * v; }`
and it is used as follows:
`constexpr auto default_address_range = 64_kb;`

I can imagine a set of these helper routines; _KB, _MB, _GB, _TB

Again something to think about.

Reply:
`64_kb` means 64 kb converted into bytes?

Reply:
Also, we have `XBU::string_to_bytes("2G")` which is pretty readable

Reply:
But very arcane.   `2_gb` is more readable and much more efficient.